### PR TITLE
v1.4 enhance ui

### LINKED ISSUE #210: Update UI
Fix UI issues.
Add animation to UI.

### PR DESCRIPTION
**Pull request by yican95**
Closes #210

### CODE REVIEW

File: src/main/java/seedu/restaurant/ui/MainWindow.java [hunk 1]
```diff
@@ -199,6 +204,14 @@ void fillInnerParts() {
...
         reservationListPanel = new ReservationListPanel(logic.getFilteredReservationList());
         personListPanelPlaceholder.getChildren().add(personListPanel.getRoot()); // Show restaurant book
+
+        ftListPanel = new FadeTransition(Duration.millis(150), personListPanelPlaceholder);
+        ftListPanel.setFromValue(0);
+        ftListPanel.setToValue(1);
+
+        ftStackPanel = new FadeTransition(Duration.millis(150), browserPlaceholder);
+        ftStackPanel.setFromValue(0);
+        ftStackPanel.setToValue(1);
```
Comment (AZhiKai) [inline src/main/java/seedu/restaurant/ui/MainWindow.java]: These 3 lines can be extracted into a method with 2 parameters maybe?

```
public void setFadeTransition(duration, placeholder) { 
... = new FadeTransition(....);
...setFromValue
...setToValue
}
```

**Comment by AZhiKai** [comment]
Also, can show an example of how the animation looks like? You can do a screen recording using ShareX.

File: src/main/java/seedu/restaurant/ui/MainWindow.java (hunk 1 above)
Comment (yican95) [reply src/main/java/seedu/restaurant/ui/MainWindow.java]: Done

**Comment by yican95** [comment]
ft

**Pull request merged**